src/instrument.js

```diff
@@ -70,7 +70,7 @@ function computeNextEntry(reducer, action, state, error, replaying) {
   let nextState = state;
   let nextError;
   try {
-    nextState = reducer(state, action, replaying);
+    nextState = reducer(state, { ...action, replaying });
```

Comment:
I am in favour of keeping the original reference untouched, of course it is a performance penalty.